sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -122,7 +128,36 @@ bool rom_test_main(void) {
     LOG_INFO("Jitter is enabled");
   }
 
+#if !OT_IS_ENGLISH_BREAKFAST
+  // Check the otp to see if flash scramble should be enabled.
+  CHECK_DIF_OK(dif_otp_ctrl_init(
+      mmio_region_from_addr(TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR), &otp_ctrl));
```

Comment:
Nice! OTP init is in the test_rom now!
This is a comment for further enhancement, does not really apply to this PR :)
Do you think we can move it out of the `!OT_IS_ENGLISH_BREAKFAST` and just remove all the otp_init in chip level test?

Reply:
o you mean persist the `otp_ctrl` declaration into the tests? 
I think @alphan / @timothytrippel can comment on that better, but I'm not sure if the stack is preserved between rom and ottf (effectively the rom_ext).  So even if I declare it global here, I'm not sure what additional things we need to do to pass it on.

However declaring it always as part of ottf would certainly be doable and we could remove it from all the downstream tests.  Maybe there's an argument to do that anyways, because we have situations inside the testutil where a handle is not passed in and they're doing it locally. 

@alphan / @timothytrippel let me know if I mis-characterized this.

Reply:
Hmm I am little confused here, why do you need to call `dif_otp_ctrl_init()` and initialize a `otp_ctrl` DIF handle in the test ROM if you are just using an `abs_mmio_read32()` below?  it looks like the `otp_ctrl` DIF handle is unused in the test ROM, unless I am missing something?

Reply:
o oops, sorry that was legacy. I previously had been using a dif_routine to read, but then changed to absolute read.
Let me get rid of this.